xapi-xenopsd-xenlight.opam

```diff
@@ -0,0 +1,11 @@
+opam-version: "1.2"
+name: xapi-xenopsd-xenlight
+maintainer: "xen-api@lists.xen.org"
+authors: "xen-api@lists.xen.org"
+homepage: "https://github.com/xapi-project/xenopsd"
+build: [[ "jbuilder" "build" "-p" name "-j" jobs ]]
+build-test: [[ "jbuilder" "runtest" "-p" name "-j" jobs ]]
```

Comment:
I think we only have tests for the `xapi-xenopsd` package, so we could omit this. The `xenopsd_test` executable only depends on `xenopsd`, but not on the other packages. This means that it's testing the `xenopsd` package, so I think it's more consistent if we only run this test for the package that it's testing. Otherwise we might end up running the same test multiple times.

Reply:
There is a test file for xenlight, but at the moment I don't think we run it. Ok, I'll remove it and we can re-implement it in future if we decide to run this test.